spec.bs

```diff
@@ -491,7 +498,7 @@ A [=component=] has an associated <dfn for=component>has regexp groups</dfn>, a
 </div>
 
 <div algorithm>
-  To perform a <dfn export for=URLPattern>match</dfn> given a {{URLPattern}} |urlpattern|, a {{URLPatternInput}} |input|, and an optional string |baseURLString|:
+  To perform a <dfn export for=URLPattern>match</dfn> given a [=URL Pattern=] |urlpattern|, a {{URLPatternInput}} |input|, and an optional string |baseURLString|:
```

Comment:
The `for=""` value should change to `for="URL pattern"` since it no longer applies to `URLPattern` instances.

Reply:
done.